plugins/config/lib/configService.js

```diff
@@ -2265,8 +2372,10 @@ function ConfigService(context) {
     next();
   });
 
+  
+
+  router.use(bodyParser.raw({type: binaryTypeCheck, limit:'3mb'}));
```

Comment:
Should add comment describing whats going on here, especially about the default value of 100kb

Reply:
Should put '3mb' as a constant at the top of the file